### PR TITLE
Create 3.1create-cluster-with-kubectl.md

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

File: docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+## Scaling clusters
+
+You can modify the value of `replicas` in `apps_v1alpha1_nebulacluster.yaml` to scale a Nebula Graph cluster.
+
+### Scale up clusters
+
+The following shows how to scale up a Nebula Graph cluster by changing the number of Storage services to 5:
+
+1. Change the value of `storaged.replicas` in `apps_v1alpha1_nebulacluster.yaml` from `3` to `5`.
```
Comment (abby-cyber) [inline docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md:134]: the value of the ``

File: docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+  nebula-storaged-2   1/1     Running   0          2ms
+  nebula-storaged-3   1/1     Running   0          5m
+  nebula-storaged-4   1/1     Running   0          5m
+  ```
+
+  As you can see above, the number of Storage services is scaled up to 5.
+
+### Scale down clusters
+
+The principle of scaling down a cluster is the same as scaling up a cluster. You scale down a cluster if the numeric value of `replicas` in `apps_v1alpha1_nebulacluster.yaml` is changed smaller than the current number. For more information, see the **Scale up clusters** section above.
```
Comment (abby-cyber) [inline docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md:184]: the numeric value of the `replicass`

File: docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+  Output:
+
+  ```bash
+  NAME                READY   STATUS    RESTARTS   AGE
+  nebula-graphd-0     1/1     Running   0          2m
+  nebula-metad-0      1/1     Running   0          2m
+  nebula-storaged-0   1/1     Running   0          2m
+  nebula-storaged-1   1/1     Running   0          2m
+  nebula-storaged-2   1/1     Running   0          2ms
```
Comment (izhuxiaoqing) [inline docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md:175]:
```suggestion
  nebula-storaged-2   1/1     Running   0          2m
```

File: docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+  nebula-storaged-3   1/1     Running   0          5m
+  nebula-storaged-4   1/1     Running   0          5m
+  ```
+
+  As you can see above, the number of Storage services is scaled up to 5.
+
+### Scale down clusters
```
Comment (izhuxiaoqing) [inline docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md:182]: ditto

**Review by cooper-lzy — APPROVED** [review]

**Review by foesa-yang — APPROVED** [review]

**Pull request merged**